.gitignore

```diff
@@ -172,3 +172,47 @@ UpgradeLog*.htm
 
 # Microsoft Fakes
 FakesAssemblies/
+src/.vs/BerconNoise/v16/Browse.VC.db
```

Comment:
Could you just ignore everything below .vs/ + .vscode/ folders, to avoid so many entries in .gitignore?